pages/common/trivy.md

```diff
@@ -1,20 +1,36 @@
 # trivy
 
 > Scanner for vulnerabilities in container images, file systems, and Git repositories, as well as for configuration issues.
-> More information: <https://github.com/aquasecurity/trivy>.
+> More information: <https://aquasecurity.github.io/trivy>.
 
-- Scan an image:
+- Scan a Docker image for vulnerabilities and exposed secrets:
```

Comment:
```suggestion
- Scan an image for vulnerabilities and exposed secrets:
```

Only docker images or also other kinds of images?

Reply:
> Only docker images or also other kinds of images?

Docker image only when subcommand is `image`. Reference here: https://aquasecurity.github.io/trivy/v0.49/docs/target/container_image/